src/sumo_backend/utils.clj

```diff
@@ -0,0 +1,227 @@
+(ns sumo-backend.utils)
+(require '[sumo-backend.mysql :as db])
+
+;; Note on ranks
+;; Komusubi, Maegashira, and Juryo are not the same
+;; Difference between them is the Maegashira's number
+;; Or all Maegashira numbers + the Juryo number
+;; e.g. Maegashira #1 is one rank away from Komusubi
+
```

Comment:
I think this part got repeated from `sumo-backend.compare`
The rank int<->str conversions should all be in the same namespace.

Reply:
gotcha, I had it in all the files as an overall note, but will only put it the namespace dealing with rank conversion 👍🏼 . The rank conversion definitely threw me for a bit